packages/testlab/README.md

```diff
@@ -2,10 +2,19 @@
 
 A collection of test utilities we use to write LoopBack tests.
 
-## `expect`
+## Overview
 
-[Should.js](https://shouldjs.github.io/) configured in "as-function" mode
-(global `Object.prototype` is left intact) with an extra chaining word `to`.
+Test utilities to help writing loopback-next tests:
+- Should.js: expect(actual, expected, [message]).
+- sinon: Standalone test spies, stubs and mocks, that Works with any unit testing framework.
+
+## Installation
+
+```
+$ npm install --save @loopback/testlab
```

Comment:
This is module is almost always used for tests only, I think we should be telling users to use `--save-dev`.

@crandmck any objections?